<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="Surgical_Tool_Tracker" pkg="myproject" type="Surgical_Tool_Tracker" output="screen" >
  </node>
	
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/Surgical_Tool_Tracker/output_video"/>
    <param name="autosize" value="true" />
  </node>

</launch>
